AsyncDisplayKit/ASImageNode+AnimatedImage.mm

```diff
@@ -85,6 +85,23 @@ - (void)coverImageCompleted:(UIImage *)coverImage
   }
 }
 
+- (NSString *)animatedImageRunLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+  return _animatedImageRunLoopMode;
+}
+
+- (void)setAnimatedImageRunLoopMode:(NSString *)runLoopMode
+{
+  ASDN::MutexLocker l(_displayLinkLock);
+
+  if (_displayLink != nil) {
+    [_displayLink removeFromRunLoop:[NSRunLoop mainRunLoop] forMode:_animatedImageRunLoopMode];
+    [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:runLoopMode];
```

Comment:
@ejensen yes, I tend to agree. Let's change the default to the more feature rich option - spending cycles on the animation during scrolling. Even from a user perspective, it would be understandable if an older or mid-level device began dropping frames during scroll tracking if it were visibly animating images.